docs/conftest.py

```diff
@@ -1,14 +1,18 @@
+import sys
 from doctest import ELLIPSIS
+
 from sybil import Sybil
 from sybil.parsers.codeblock import CodeBlockParser
 from sybil.parsers.doctest import DocTestParser, FIX_BYTE_UNICODE_REPR
 from sybil.parsers.skip import skip
 
-pytest_collect_file = Sybil(
-    parsers=[
-        DocTestParser(optionflags=ELLIPSIS | FIX_BYTE_UNICODE_REPR),
-        CodeBlockParser(future_imports=['print_function']),
-        skip
-    ],
-    pattern='index.rst',
-).pytest()
+if sys.version_info >= (3, 6):
+    pytest_collect_file = Sybil(
+        parsers=[
+            DocTestParser(optionflags=ELLIPSIS | FIX_BYTE_UNICODE_REPR),
+            CodeBlockParser(future_imports=['print_function']),
+            skip
+        ],
+        pattern='index.rst',
+        excludes=['docs/tutorials/dcdi/* ']
```

Comment:
can you add a comment about why to include or exclude something?

Reply:
First, sorry to say that I pushed a messed-up merge. I am going to do a better job in the future making sure everything passes in the Travis of my fork.

On this point, the excludes was an experiment that didn't work. I should have taken it out. I will do so now. It's related to a ticket I filed with Sybil, the worked-around by wrapping the block in a test for Python 3.7.